java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/models/healthData/RecordExportStatusRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.sagebionetworks.bridge.sdk.models.healthData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.sdk.Config;
+import org.sagebionetworks.bridge.sdk.models.upload.UploadCompletionClient;
+import org.sagebionetworks.bridge.sdk.models.upload.UploadStatus;
+
+import java.util.List;
+
+/** Represents a record export status request to the Bridge Server. */
+public class RecordExportStatusRequest {
+
+    private List<String> recordIds;
+    private Config.ExporterStatus synapseExporterStatus;
+
+    public List<String> getRecordIds() { return this.recordIds; }
+
+    public Config.ExporterStatus getSynapseExporterStatus() { return this.synapseExporterStatus; }
```

Comment:
I would pull this constant out to the back where RecordExportStatusRequest resides, this is the only enum in Config and that's inconsistent.